docs/maps/search.asciidoc

```diff
@@ -35,13 +36,29 @@ image::maps/images/global_search_bar.png[]
 [[maps-create-filter-from-map]]
 === Create filters from a map
 
-You can create two types of filters by interacting with your map:
+You can create three types of filters by interacting with your map:
 
-* <<maps-spatial-filters, Spatial filters>>
```

Comment:
The intro needs a little work.  Maybe something like this:

Add filters to your map that allow you to .....  **Maps** provides three filters:

- Map extent filters
- Spatial filters
- Phrase filters

Can you make the bulleted list parallel?